src/built-in-definitions.js

```diff
@@ -0,0 +1,108 @@
+export const definitions = {
```

Comment:
This is basically the opposite of built-in-features.js so maybe can just reverse it somehow